### PR TITLE
Add clangarm64 to OpenBLAS

### PR DESCRIPTION
**Pull request by hmartinez82**

### CODE REVIEW

**Comment by lazka** [comment]
lgtm, thanks

File: mingw-w64-openblas/PKGBUILD [hunk 1]
```diff
@@ -72,6 +75,13 @@ _build_openblas() {
     _c_lapack_opt="-DC_LAPACK=ON"
   fi
 
+  declare target=CORE2
+  declare dynamic_arch="-DDYNAMIC_ARCH=ON"
+  if [[ ${MINGW_PACKAGE_PREFIX} == *-aarch64 ]]; then
+    target=CORTEXA53
+    dynamic_arch="-DDYNAMIC_ARCH=OFF"
```
[the remark this answers is not in the document]
Comment (hmartinez82) [reply mingw-w64-openblas/PKGBUILD:82]: @lazka I tried to build with dynamic arch enabled, but clang wasn't able to compile all the kernels inside he arm64 folder. Do you know if there's a way to enable dynamic arch, but limit the kernels. Windows on ARM laptops today out there only have CortexA53 and CortexA57 (in their efficient cores).

[the remark this answers is not in the document]
Comment (lazka) [reply mingw-w64-openblas/PKGBUILD:82]: I was just asking so the next person looking at it knows why it's disabled.

According to the readme it's supported in theory for aarch64, but likely only tested on Unix.

[the remark this answers is not in the document]
Comment (lazka) [reply mingw-w64-openblas/PKGBUILD:84]: I added an inline comment

**Pull request merged**